code/modules/research/rdconsole.dm

```diff
@@ -218,10 +218,8 @@ doesn't have toxins access.
 	var/list/l = list()
 	l += "<h2>Nanotrasen Research and Development</h2>[RDSCREEN_NOBREAK]"
 	l += "<div class='statusDisplay'><b>Connected Technology database: [stored_research == SSresearch.science_tech? "Nanotrasen" : "Third Party"]"
-	l += "Available Points: [stored_research.research_points]"
+	l += "Available Research Points: [stored_research.research_points]"
 	l += "Security protocols: [emagged? "<font color='red'>Disabled</font>" : "<font color='green'>Enabled</font>"]"
-	l += "Design Disk: [d_disk? "<font color='green'>Loaded</font>" : "<font color='red'>Not Loaded</font>"] | \

```

Comment:
Uhhhh what? No. These are here for a reason, disks are still used.

Reply:
Since my last PR the options to use the disks don't appear until a disk is inserted, it still conveys the same information. The original menu was EXTREMELY bloated with all sorts of redundant or irrelevant information.

Reply:
No it doesn't this makes sure people know disks /exist/

Reply:
Same way people figure out 99% of shit in this game and its hardly a core mechanic. Once you put the disk inside its all self explanatory.

Reply:
i don't see how an extra line indicating they can receive tech/design disks is causing so many problems, what's the point of this?